core/lib/snt/core/services/event.rb

```diff
@@ -2,32 +2,103 @@ module SNT
   module Core
     module Services
       class Event
-        attr_accessor :hotel, :object, :action_type, :details, :published
+        ##### Event trigger with initial event information to +service+.dto queue
+        #
+        # e.g.: ::SNT::Core::MQ.publisher.publish(+event_context+, routing_key: 'pms.dto')
+        #
+        ### Case #1: Normal Event with Object and attributes
+        #
+        #   event_args = {
+        #     service: :pms,
```

Comment:
Can you define what all of the different args are and what naming conventions should be used?

Reply:
Add descriptions for each arguments above the event_context method